### ISSUE TITLE
[qt5-base] build failure on macOS 10.15

### ISSUE THREAD

**Issue by FranciscoPombal** [microsoft/vcpkg#12770]
**Host Environment**
- OS: macOS 10.15 from GitHub Actions
- Compiler: whichever one is included in XCode 11.6

**To Reproduce**
`./vcpkg install qt5-base:x64-osx-release --clean-after-build --overlay-triplets=triplets_overlay`

Contents of `triplets_overlay/x64-osx-release.cmake`:

```cmake
set(VCPKG_TARGET_ARCHITECTURE x64)
set(VCPKG_CRT_LINKAGE dynamic)
set(VCPKG_LIBRARY_LINKAGE static)

set(VCPKG_CMAKE_SYSTEM_NAME Darwin)
set(VCPKG_BUILD_TYPE release)
```

**Failure logs** 
Check out the output of the "post-mortem" step here (it's the output of the following commands - `cat /Users/runner/qbt_tools/buildtrees/qt5-base/config-x64-osx-release-rel-out.log` and `cat /Users/runner/qbt_tools/buildtrees/qt5-base/config-x64-osx-release-rel-err.log`):
https://github.com/FranciscoPombal/qBittorrent/runs/946554718?check_suite_focus=true

**Additional context**

Looks like it fails in the configure step:

```
(...)
Configure summary:

Build type: macx-clang (x86_64, CPU features: cx16 mmx sse sse2 sse3 ssse3 sse4.1)
Compiler: clang (Apple) 11.0.3
Configuration: sse2 aesni sse3 ssse3 sse4_1 sse4_2 avx avx2 avx512f avx512bw avx512cd avx512dq avx512er avx512ifma avx512pf avx512vbmi avx512vl compile_examples f16c largefile precompile_header rdrnd rdseed shani x86SimdAlways release c++11 c++14 c++17 c++1z concurrent dbus no-pkg-config reduce_exports static stl
Build options:
(...)
Qt Sql Drivers:
  DB2 (IBM) .............................. no
  InterBase .............................. no
  MySql .................................. no
  OCI (Oracle) ........................... no
  ODBC ................................... no
  PostgreSQL ............................. yes
  SQLite2 ................................ no
  SQLite ................................. yes
    Using system provided SQLite ......... yes
  TDS (Sybase) ........................... no
Qt Testlib:
  Tester for item models ................. yes

Note: Using static linking will disable the use of dynamically
loaded plugins. Make sure to import all needed static plugins,
or compile needed modules into the library.

ERROR: Feature 'system-harfbuzz' was enabled, but the pre-condition 'features.harfbuzz && libs.harfbuzz' failed.
```

I don't have a macOS system so I can't test this further on my own.

**Comment by JackBoosY** [comment]
@Neumann-A Could you please take a look?

Since this is an unofficial triplet, we will not fix this issue.

**Comment by Neumann-A** [comment]
```
2020-08-04T20:58:06.9544730Z > ld: warning: directory not found for option '-L/Users/runner/qbt_tools/installed/x64-osx-release/lib/manual-link'
2020-08-04T20:58:06.9544870Z > Undefined symbols for architecture x86_64:
2020-08-04T20:58:06.9544980Z >   "_BrotliDecoderDecompress", referenced from:
2020-08-04T20:58:06.9545080Z >       _woff2_open_font in libfreetype.a(sfnt.c.o)
2020-08-04T20:58:06.9545180Z > ld: symbol(s) not found for architecture x86_64
2020-08-04T20:58:06.9545650Z > clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

should be fixed by #12405

**Comment by FranciscoPombal** [comment]
Thanks, I can confirm it works now.
